modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -75,13 +80,14 @@ class ImagingUploader extends React.Component {
                 Module="imaging_uploader"
                 name="imaging_filter"
                 id="imaging_filter"
+                ref="imagingUploaderFilter"
```

Comment:
string refs are being deprecated. use a callback function similar to [what was done here for data dictionary](https://github.com/aces/Loris/pull/3432/files#diff-b7cfdf269c10c4ab50f7710f6cfdaee7R81)